jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/ContextHandler.java

```diff
@@ -80,13 +80,25 @@ public class ContextHandler extends Handler.Wrapper implements Attributes, Grace
      * Get the current Context if any.
      *
      * @return The {@link Context} from a {@link ContextHandler};
-     *         or {@link Server#getContext()} if the current {@link Thread} is not scoped to a {@link ContextHandler}.
+     *         or null if the current {@link Thread} is not scoped to a {@link ContextHandler}.
      */
     public static Context getCurrentContext()
```

Comment:
Seems like we should get rid of the no-arg `getCurrentContext` method then, as it wasn't correct to call it in the circumstances you've fixed, but there might be more lurking in the code?